File: 908/SmallestRangeI.scala
object SmallestRangeI {
    def smallestRangeI(A: Array[Int], K: Int): Int = (A.max - A.min - 2*K) max 0

    def main(args: Array[String]) = {
        val A = Array(0, 10)
        val K = 2
        println(smallestRangeI(A, K))
    }
}

File: 908/smallest-range-i.js
/**
 * @param {number[]} A
 * @param {number} K
 * @return {number}
 */
var smallestRangeI = function (A, K) {
    return Math.max(Math.max(...A) - Math.min(...A) - 2 * K, 0);
};